From a5323b98697a5b1c6f74da88f28c4d17b46b3e31 Mon Sep 17 00:00:00 2001
Message-ID: <a5323b98697a5b1c6f74da88f28c4d17b46b3e31.1746781039.git.lorenzo@kernel.org>
In-Reply-To: <b840c5d58386d4600544ec7e02c2c3e19ce96bc6.1746781039.git.lorenzo@kernel.org>
References: <b840c5d58386d4600544ec7e02c2c3e19ce96bc6.1746781039.git.lorenzo@kernel.org>
From: Peter Chiu <chui-hao.chiu@mediatek.com>
Date: Mon, 30 Oct 2023 20:19:41 +0800
Subject: [PATCH 029/148] mtk: mt76: mt7996: enable ser query

Do not return -EINVAL when action is UNI_CMD_SER_QUERY for user
to dump SER information from FW.

Signed-off-by: Peter Chiu <chui-hao.chiu@mediatek.com>
---
 mt7996/mcu.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/mt7996/mcu.c b/mt7996/mcu.c
index 349c69e5..e97fff8b 100644
--- a/mt7996/mcu.c
+++ b/mt7996/mcu.c
@@ -3923,6 +3923,8 @@ int mt7996_mcu_set_ser(struct mt7996_dev *dev, u8 action, u8 val, u8 band)
 	};
 
 	switch (action) {
+	case UNI_CMD_SER_QUERY:
+		break;
 	case UNI_CMD_SER_SET:
 		req.set.mask = cpu_to_le32(val);
 		break;
-- 
2.49.0

